tests/migration_test.go

```diff
@@ -692,6 +686,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				// Start a ISCSI POD and service
 				By("Starting an iSCSI POD")
 				iscsiIP := tests.CreateISCSITargetPOD(cd.ContainerDiskEmpty)
+				tests.SkipIscsiPVCWithIPv6Test(iscsiIP)
```

Comment:
Let's wait with this PR till 
https://github.com/kubevirt/project-infra/pull/642
https://github.com/kubevirt/kubevirt/pull/4156
are merged.
The we can assume there is not single stack ipv6 lane. And we can use [libnet.GetIp](https://github.com/kubevirt/kubevirt/pull/4156/files#diff-e96bc4003adf459cfc269a13a00c944bR8) to get the ipv4 address of the pod. I prefer it over assuming the podip is ipv4 one.

Reply:
I dont think we should wait for #4156, until #4150 is resolved there is no point in changing `CreateISCSITargetPod`.
Why assuming that podIP is ipv4 is wrong?

Reply:
its only a limitation atm, 
https://github.com/containerd/cri/issues/1278
at the future the primary might be ipv6

Reply:
imo if you dont want to wait you can dual the tests, and add a skip on ipv6 part
this way you wont be depended on nothing, 
but you shouldnt assume the order please, and you need to use DescribeTable as its the new method we should use unless we decide otherwise.

Reply:
I would agree with you if this test would work using IPv6 address for that ISCSI PV, but it doesn't so I dont think we should waste time adapting a test that is not going to work.